src/components/PersonLink.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Person } from '../types/Person';
+import { Link } from 'react-router-dom';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  const isFemale = person.sex === 'f';
```

Comment:
```suggestion
  const isFemale = person.sex === 'f';
```

what is `f`? looks like magic value, move to constants